docs/INSTALL.md

```diff
@@ -285,6 +285,27 @@ dictionary:
     - nato.int
     - dds.mil
     - "*.isaf.nato.int"
+  imagery:
+    mapOptions:
+      crs: EPSG3857
+    baseLayers:
+      - name: OSM
+        default: true
+        type: tile
+        url: "http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      - name: World Imagery Tiles
+        default: false
+        type: tile
+        url: "http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}"
+        options:
+          tms: false
+      - name: World WMS
+        default: false
+        type: wms
+        url: "https://www.gebco.net/data_and_products/gebco_web_services/web_map_service/mapserv"
+        options:
+          layer: GEBCO_LATEST
```

Comment:
See previous remarks about `https` and `layers`. Probably use `https` everywhere in the docs, as in a real deployment environment this most likely *has* to be `https` because the application will run behind `https` (and then the browser will refuse to load data from http on a secure page).